gapii/client/adb.go

```diff
@@ -72,6 +72,23 @@ func Start(ctx context.Context, p *android.InstalledPackage, a *android.Activity
 		abi = p.Device.Instance().GetConfiguration().PreferredABI(nil)
 	}
 
+	driver, err := d.GraphicsDriver(ctx)
```

Comment:
What's returned here does not necessarily match what we use in `SetupPrereleaseDriver` function. We should have a separate function that returns the prerelease driver apk without the override. The override should be removed or renamed in the future because it's misleading now